portfolio-project-matching-app/pages/browseProjects.js

```diff
@@ -30,11 +34,11 @@ const browseProjects = () => {
         // tracks whether component mounted, cleanup will assign false
         let isMounted = true
         // get projects and set state if component mounted
-        getAllProjects().then((projects) => {
+        readAllDocs('projects').then((projects) => {
             if (isMounted) setVisibleProjects(projects)
         })
         // get technologies and set state if component mounted
-        getAllTechnologies().then((technologies) => {
+        readAllDocs('technologies').then((technologies) => {
```

Comment:
Using these new "lightweight" functions instead of `getAllProjects()` and `getAllTechnologies()`